.github/workflows/build.yml

```diff
@@ -132,12 +154,16 @@ jobs:
       # Build All Packages
       - name: pack experiments
         working-directory: ./tooling/Scripts/
-        run: ./PackEachExperiment.ps1 all
-
-      # Push Packages to our DevOps Artifacts Feed
-      - name: Add source
-        if: ${{ github.ref == 'refs/heads/main' }}
-        run: dotnet nuget update source LabsFeed --username dummy --password ${{ secrets.DEVOPS_PACKAGE_PUSH_TOKEN }}
```

Comment:
@Arlodotexe this is still needed in between the added PR step and the 'Push packages' still. This is why `main` is failing, as the source isn't updated to include the token.